nature_importer.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+import argparse
+import csv
+import os
+import importer_utils as utils
+import wikidataStuff.wdqsLookup as lookup
+from NatureArea import NatureArea
+from PreviewTable import PreviewTable
+from Uploader import Uploader
+
+DATA_DIRECTORY = "data"
+reserves_file = "NR_polygon.csv"
+nationalparks_file = "NP_polygon.csv"
+edit_summary = "test"
+
+
+def get_data_from_csv_file(filename):
+    """Load data from csv file into a list."""
+    with open(filename, "r") as f_obj:
+        reader = csv.DictReader(f_obj, delimiter=',')
+        csv_data = list(reader)
+    return csv_data
+
+
+def remove_duplicate_entries(nature_dataset):
+    """
+    Remove duplicate entries from Nature Reserve file.
+
+    In cases where a reserve occurs twice, it's because
+    of different values of "BESLSTATUS". Whenever that happens,
+    one of them is always "gällande". This is the one we keep,
+    and the other one is removed.
+    """
+    results = []
+    unique_ids = []
+    for row in nature_dataset:
+        n_id = row["NVRID"]
+        if n_id in unique_ids:
```

Comment:
Since we (presumably) know that one will be "gällande" and one won't we could simply do:
```python
new_status = row["BESLSTATUS"]
if new_status == "Gällande":
    utils.remove_dic_from_list_by_value(results, "NVRID", n_id)
    results.append(row)
```

See also previous comment.